File: Google.Apis.Sample.MVC/Models/TimeReport.cs
namespace Google.Apis.Sample.MVC.Models
{
    public class TimeReport
    {
        public string Date { set; get; }
        public string Project { set; get; }
        public string Task { set; get; }
        public string Type { set; get; }
        public string Description { set; get; }
        public decimal Duration { set; get; }
        public decimal Overtime { set; get; }
    }
}

File: Google.Apis.Sample.MVC/Scripts/App/TimeReport.js
function timeReport() {
    var self = this;
    self.Date = ko.observable("");
    self.Project = ko.observable("");
    self.Task = ko.observable("");
    self.Type = ko.observable("");
    self.Description = ko.observable("");
    self.Duration = ko.observable(0);
    self.Overtime = ko.observable(0);
};